ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -119,14 +119,14 @@ fi
 if [ ${Z_VERSION} -lt 11 ]; then
   MACHINE_POOL_OVERRIDES="controlPlane:
   name: master
-  replicas: 3
+  replicas: ${CONTROL_PLANE_REPLICAS}
```

Comment:
seems the update is not enough, the customization is only applied on ocp version less than 4.11 here.

Reply:
yes, i notice that. i will add an env :${CUSTOMIZE_PHYSICAL_RESOURCES}"  for customization setting in another pr.

Reply:
i guess i no need update hardcode replicas here. just add an env, if CUSTOMIZE_PHYSICAL_RESOURCES == yes; then. set replicas, cpu, memory..disksize to customization. WDYT.

Reply:
Why we need to introduce another env, personally it is possible to use the existing `CONTROL_PLANE_REPLICAS` to complete the function in this PR.

Reply:
i'm not sure if there any changes when z_version great than 11. is that change looks good to you? @jinyunma

Reply:
ok. so if not specific disksize or memory. it also have default value right? if that case, then i think it's ok to me.